### ISSUE TITLE
The minimal-app is broken in Debug mode

### ISSUE THREAD

**Issue by hoshiryu** [STORM-IRIT/Radium-Engine#356]
When running the minimal-app (helloRadium) from master-v1, the app crashes with the following output:
```
- 02:25:30 PM INFO: *** Radium Engine ***
- 02:25:30 PM WARNING: Binding action 8 to code 1, which is already used for action 2.
- 02:25:30 PM WARNING: Binding action 11 to code 1, which is already used for action 2.
- 02:25:30 PM INFO: *** Radium Engine OpenGL context ***
- 02:25:30 PM INFO: Renderer (glbinding) : GeForce GTX 1080/PCIe/SSE2
- 02:25:30 PM INFO: Vendor   (glbinding) : NVIDIA Corporation
- 02:25:30 PM INFO: OpenGL   (glbinding) : 4.4
- 02:25:30 PM INFO: GLSL                 : 4.40 NVIDIA via Cg compiler
- 02:25:30 PM INFO: *** Radium Engine Viewer ***
- 02:25:30 PM DEBUG: Loading shader Shaders/Default.vert.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Default.frag.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Basic2D.vert.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/DrawScreen.frag.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Basic2D.vert.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/DrawScreenI.frag.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Basic2D.vert.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/CircleBrush.frag.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Picking.vert.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Picking.frag.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/PickingPoints.geom.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Picking.vert.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/Picking.frag.glsl
- 02:25:30 PM DEBUG: Loading shader Shaders/PickingLines.geom.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Picking.vert.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Picking.frag.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/PickingLinesAdjacency.geom.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Picking.vert.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Picking.frag.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/PickingTriangles.geom.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Basic2D.vert.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/FinalCompose.frag.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Plain.vert.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/UnlitOIT.frag.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Basic2D.vert.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/ComposeOIT.frag.glsl
- 02:25:31 PM INFO: Loaded : DefaultLightManager
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM ERROR: FBO Error : GL_FRAMEBUFFER_INCOMPLETE_ATTACHMENT
- 02:25:31 PM INFO: [Viewer] Set active renderer: Forward Renderer
- 02:25:31 PM INFO: Loaded : Minimal system
- 02:25:31 PM DEBUG: Loading shader Shaders/Materials/BlinnPhong/BlinnPhong.vert.glsl
- 02:25:31 PM DEBUG: Loading shader Shaders/Materials/BlinnPhong/BlinnPhong.frag.glsl
- 02:25:31 PM ERROR: OpenGL error (/ssd/canezin/Radium_PR/Radium-Engine/src/Engine/Renderer/Renderers/ForwardRenderer.cpp:145, glClearBufferfv( GL_COLOR, 0, clearColor.data() )) :  Invalid framebuffer operation : The framebuffer object is not complete. The offending command is ignored and has no other side effect than to set the error flag.
(GL_INVALID_FRAMEBUFFER_OPERATION : 0xGL_INVALID_FRAMEBUFFER_OPERATION).
The program has unexpectedly finished.
```

**Comment by dlyr** [comment]
I haven't this bug, but nothing on screen :(

**Comment by dlyr** [comment]
well after a resize, the cube appear... maybe it's related.

**Comment by MathiasPaulin** [comment]
The helloRadium app is working in all modes (Release, Debug) since master-v1 refactoring. Issue closed.

**Comment by MathiasPaulin** [comment]
I can't reproduce the issue. As you have identified the problem, you can fix it and make a PR. Do it as soon as possible so that the release of Master V1 will not be delayed due to this issue.

**Comment by MathiasPaulin** [comment]
This issue might be solved by PR #448. @hoshiryu could you test and close this issue if solved ?
